packages/components/src/components/notification/notification.stories.ts

```diff
@@ -0,0 +1,48 @@
+export default {
+  title: 'Components/Notification',
+  tags: ['autodocs'],
+
+  args: {
+    content: "Sample Notification",
+    variant: 'success',
+    icon: 'c-check-16',
```

Comment:
Refer to the Button story to enable the user to choose an icon from a dropdown list instead of setting it as a string